pkg/config.go

```diff
@@ -39,6 +39,37 @@ func (*Config) Project() string {
 	return os.Getenv("GOOGLE_PROJECT")
 }
 
+// TlsVersion will be used current TLS Policy
+// by this value. The acceptable value MUST BE TLS 1.0/1.1/1.2/1.3
+// Note that the TLS 1.3 supports `TLS_AES_128_GCM_SHA256`,
+// `TLS_AES_256_GCM_SHA384` and `TLS_CHACHA20_POLY1305_SHA256` ciphers only.
+// Default is "TLS_1_2"
+func (*Config) TlsVersion() string {
+	allow_vals := []string{ "TLS_1_0", "TLS_1_1", "TLS_1_2", "TLS_1_3"}
+	for _, val := range allow_vals {
+        if os.Getenv("MIN_TLS_VERSION") != val || os.Getenv("MIN_TLS_VERSION") == "" {
+          fmt.Println("Input TLS version is not supported")
+					return "TLS_1_2"
+        }
+    }
+	return os.Getenv("MIN_TLS_VERSION")
```

Comment:
Indenation doesn't look uniform. I don't see the Println is necessary if it just returns a value anyway, also the code should be reduced in repeating things.

Reply:
Fixed byhttps://github.com/commercetools/gcp-ssl-policy-asserter/pull/3/commits/bf7502003062ca1233ede4bccd62e9f4d4429d8b